WebAPI/Services/BankStatmentReader.cs

```diff
@@ -125,13 +121,14 @@ private decimal GetAmout(decimal sum, string debitOrCredit)
             /// Įmokos kodas,Mokėjimo paskirtis,Kitos pusės BIC,Kitos pusės Kredito įstaigos pavadinimas,Kitos pusės Sąskaitos Nr.,
             /// Kitos pusės Pavadinimas,Kitos pusės Asmens kodas/Registracijos Nr.,Kitos pusės Kliento kodas mokėtojo informacinėje sistemoje
 
-            return csv.TryGetField("Orig. suma", out decimal sum) && csv.TryGetField("C/D", out string debitOrCredit)
+
+            return  csv.TryGetField("Orig. suma", out string debitOrCredit) && csv.TryGetField("Ekvivalentas", out string suma) && csv.TryGetField("Laikas", out string data)
                 ? new Transaction
                 {
-                    TrTime = csv.GetField<DateTime>("Data"),
-                    Amount = GetAmout(sum, debitOrCredit),
-                    CounterParty = csv.GetField("Kitos pusės Pavadinimas"),
-                    Details = csv.GetField("Mokėjimo paskirtis")
+                    TrTime = DateTime.ParseExact(data, "yyyyMMdd", CultureInfo.InvariantCulture),
+                    Amount = GetAmout(Convert.ToDecimal(suma, CultureInfo.InvariantCulture), debitOrCredit),
+                    CounterParty = csv.GetField("Kitos pus�s S�skaitos Nr."),
+                    Details = csv.GetField("Kitos pus�s BIC")
```

Comment:
What are those question marks purpose?

Reply:
There should be Lithuanian letters but csv file of SEB LT doesn't show them correctly

Reply:
It's an encoding problem. It's possible that file you are testing on, has incorrect encoding or was encoded incorrectly. Once user will upload real bank statement from a bank it will contain all lithuanian letters and Reader gonna fail.